posthog/settings.py

```diff
@@ -74,9 +74,10 @@ def print_warning(warning_lines: Sequence[str]):
 else:
     JS_URL = os.getenv("JS_URL", "")
 
+DISABLE_MMDB = get_from_env("DISABLE_MMDB", False, type_cast=strtobool)  # plugin server setting disabling GeoIP feature
```

Comment:
This doesn't follow naming convention from other plugins env variables. How about `PLUGINS_DISABLE_MMDB`?

Reply:
Nope nope, this is an existing plugin server setting, so that if you run `DISABLE_MMDB=1 ./bin/start` the single setting has the appropriate effect in both Python and Node.